redhat/varnish_reload_vcl

```diff
@@ -75,8 +67,20 @@ else
 	secret="-S $VARNISH_SECRET_FILE"
 fi
 
+if [ -z "$VARNISH_ADMIN_LISTEN_ADDRESS" -a -z "$VARNISH_ADMIN_LISTEN_PORT" ]; then
+	mgmt_interface=""
+
+elif [ -n "$VARNISH_ADMIN_LISTEN_ADDRESS" -a -n "$VARNISH_ADMIN_LISTEN_PORT" ]; then
+	mgmt_interface="-T $VARNISH_ADMIN_LISTEN_ADDRESS:$VARNISH_ADMIN_LISTEN_PORT"
+
+else
+	echo "Error: only one of VARNISH_ADMIN_LISTEN_ADDRESS or VARNISH_ADMIN_LISTEN_PORT was defined"
+        echo "       Both or none must be specified"
+	exit 2
+fi
+
 # Done parsing, set up command
-VARNISHADM="varnishadm $secret -T $VARNISH_ADMIN_LISTEN_ADDRESS:$VARNISH_ADMIN_LISTEN_PORT"
+VARNISHADM="varnishadm $secret $mgmt_interface"
```

Comment:
I would simply get rid of the VARNISHADM variable and remnants of the cli/secret setup, this is meant for local service reloads so having root privileges should be enough.